File: sdk-js/examples/analyze.js
const { ChainWise, formatRiskScore } = require('../dist');

async function main() {
  const chainwise = new ChainWise();
  const result = await chainwise.analyze({
    address: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
    chain: 'ethereum',
  });

  console.log(`Risk Score: ${formatRiskScore(result.risk_score)}`);
  console.log(`Classification: ${result.classification}`);
  console.log(`Confidence: ${Math.round(result.confidence * 100)}%`);
  console.log('Layers:', result.layers);
}

main().catch((err) => {
  console.error('Analysis failed:', err.message);
  process.exit(1);
});

File: sdk-py/examples/analyze.py
from chainwise import ChainWise


def main():
    client = ChainWise()
    result = client.analyze("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48", chain="ethereum")  # USDC

    print(f"Risk Score: {result['risk_score']}/100")
    print(f"Classification: {result['classification']}")
    print(f"Confidence: {round(result['confidence'] * 100)}%")
    print("Layers:", result["layers"])


if __name__ == "__main__":
    main()
